modules/ROOT/partials/configuration/toolbar_mode.adoc

```diff
@@ -104,12 +110,14 @@ tinymce.init({
 });
 ----
 
+[[wrap]]
 ==== Wrap
 
 If the `toolbar_mode` is configured to `wrap`, the overflow toolbar buttons will be shown on one or more toolbars below the primary toolbar.
 
-image::{baseurl}/images/wrapped_toolbar.png[Wrapped Toolbar]
+image:wrapped_toolbar.png[Wrapped Toolbar]
```

Comment:
This change is incorrect as it's meant to be a block image, not an inline image.
```suggestion
image::wrapped_toolbar.png[Wrapped Toolbar]
```